src/simplewallet/simplewallet.h

```diff
@@ -342,7 +342,7 @@ namespace cryptonote
 
     //----------------- i_wallet2_callback ---------------------
     virtual void on_new_block(uint64_t height, const cryptonote::block& block);
-    virtual void on_money_received(uint64_t height, const crypto::hash &txid, const cryptonote::transaction& tx, uint64_t amount, const cryptonote::subaddress_index& subaddr_index, uint64_t unlock_time);
+    virtual void on_money_received(uint64_t height, const crypto::hash &txid, const cryptonote::transaction& tx, uint64_t amount, const cryptonote::subaddress_index& subaddr_index, bool change, uint64_t unlock_time);
```

Comment:
Minor comment:

```suggestion
    virtual void on_money_received(uint64_t height, const crypto::hash &txid, const cryptonote::transaction& tx, uint64_t amount, const cryptonote::subaddress_index& subaddr_index, bool is_change, uint64_t unlock_time);
```

The same in the other header file.

Reply:
Or maybe "bool is_change_but_false_might_still_be_change_if_out_of_date_multisig", I wonder.

Reply:
You called it `bool is_change` in the source file implementation.

Reply:
Ah. OK, I thought it was just pointless bikeshedding :P